docs/commands/status.md

```diff
@@ -1,17 +1,36 @@
 # `licensed status`
 
-The status command finds all dependencies and checks whether each dependency has a valid cached record.
+The status command finds all dependencies and checks whether each dependency has a valid record.  There are two methods for checking dependencies' statuses
+
+## Checking status with metadata loaded from cached files
+
+This is the default method for checking the status for dependencies and the recommended method for using licensed.  Checking status from cached metadata files will occur when the `--data-source` CLI flag is either unset, or set to `--data-source=files`.
+
+When using `licensed status` in this scenario, licensed will only compile a minimal amount of dependency metadata like the dependency name and version to match against cached files.  Dependency license and notice texts are loaded from cached files that are created from the [licensed cache](./cache.md) command.
 
 A dependency will fail the status checks if:
 
 1. No cached record is found
 2. The cached record's version is different than the current dependency's version
 3. The cached record's `licenses` data is empty
-4. The cached record's `license` metadata doesn't match an `allowed` license from the dependency's application configuration.
+4. The cached record's `license` metadata doesn't match an `allowed` license from the dependency's application configuration and the dependency has not been marked `reviewed` or `ignored`
    - If `license: other` is specified and all of the `licenses` entries match an `allowed` license a failure will not be logged
 5. The cached record is flagged for re-review.
    - This occurs when the record's license text has changed since the record was reviewed.
 
+## Checking status with computed metadata
+
+Checking status with computed metadata and the licensed configuration will occur when the `--data-source` CLI flag is set to `--data-source=configuration`.
+
+When using `licensed status` in this scenario, licensed will compile all dependency metadata and license text to use in status checks.  There is no need to run `licensed cache` prior to running `licensed status --data-source=configuration`.
+
+A dependency will fail the status checks if:
+
+1. The cached record's `licenses` data is empty
+2. The cached record's `license` metadata doesn't match an `allowed` license from the dependency's application configuration and the dependency has not been marked `reviewed` or `ignored`
```

Comment:
I feel like using "cached record" here might be confusing

```suggestion
1. The record's `licenses` data is empty
2. The record's `license` metadata doesn't match an `allowed` license from the dependency's application configuration and the dependency has not been marked `reviewed` or `ignored`
```

Maybe further rewording this to use the term "dependency" instead of "record" would be even better, but for me "record" suffices.

Reply:
I'm going to leave this as `record`, but 👍  to removing `cached`  The dependency is computed live on every licensed execution while the record may not be, especially in the case of using files written to a repo.

As an example, the record may have been written to reference a dependency at version 1.0.0, while the live computed dependency is at 2.0.0.  These types of mismatches can occur with all of the fields that cause a `licensed status` error.